frontend/src/metabase/css/components/buttons.module.css

```diff
@@ -8,7 +8,7 @@
   text-decoration: none;
   padding: 0.5rem 0.75rem;
   background: transparent;
-  border: 1px solid color-mod(var(--mb-color-border) blackness(5%));
+  border: 1px solid color-mix(in srgb, var(--mb-color-border), black 5%);
```

Comment:
just to be blunt. I don't think the 2 implementations are equivalent. I couldn't figure out how `blackness` and `brightness` (we don't use it tho) are implemented. I only know how `color`'s `darken` and `lighten` works. It works by increasing or decreasing the proportion of the lightness in HSL space. And this color mix, provides similar results, although not exactly the same.

I haven't tested these 2 places, but the result should be fairly similar.

Reply:
Interesting, I'm planning on testing this PR before merging today so I'll try to measure this. Will see if there's a better way of porting `blackness` to color-mix.

Reply:
by the way, I think this approach is okay, I'm just stating that the values would likely be not the same values.